README.md

```diff
@@ -191,6 +191,9 @@ The CNV case and PoN workflows (description and examples)
 1       35138   35174   target4
 ....snip....
 ```
+Before running the workflows, we recommend padding the target file by 250 bases with the ``PadTargets`` tool.  Example:  ``java -jar gatk-protected.jar --targets initial_target_file.bed --output initial_target_file.padded.bed --padding 250``
```

Comment:
Would be nice to say why padding is recommended, or what goes into determining the size should be 250.